src/components/bulk-email-tool/bulk-email-form/BulkEmailRecipient.jsx

```diff
@@ -70,6 +70,19 @@ export default function BulkEmailRecipient(props) {
             description="A selectable choice from a list of potential email recipients"
           />
         </Form.Checkbox>
+        {
+          // additional cohorts
+          additionalCohorts
+            && additionalCohorts.map((cohort) => (
+              <Form.Checkbox
+                key={cohort}
+                value={`cohort:${cohort}`}
+                disabled={selectedGroups.find((group) => group === (DEFAULT_GROUPS.AUDIT || DEFAULT_GROUPS.VERIFIED))}
+              >
+                Cohort: {cohort}
```

Comment:
Is this text displayed to the user of the tool? If so, can we make sure the string is resourced for translation? Thanks!